source/tutorial/dotnet.txt

```diff
@@ -105,58 +106,27 @@ Set up the Template App
 
       In Visual Studio, take a few minutes to explore how the solution is
-      organized. This is a standard multi-platform solution, based on a template
-      provided by Visual Studio, that has been modified for our specific use. The
-      solution has 3 projects:
-
-      .. cssclass:: config-table
-      .. list-table::
-         :header-rows: 1
-         :widths: 40 60
-
-         * - Project
-           - Purpose
-
-         * - ``realm-todo-dotnet``
-           - Provides shared resources, including the view pages and the Item model 
-             that represents the Realm object we will store in the database.
-
-             To learn more about how Realm Object Schemas are used in .NET applications,
-             see :ref:`Realm Objects <dotnet-objects>` in our .NET client guide.
-
-         * - ``realm-todo-dotnet.Android``
-           - Contains Android-specific resources, such as the Toolbar formatting.
-
-         * - ``realm-todo-dotnet.iOS``
-           - Contains iOS-specific resources.
-
-      You will perform all of the steps in this tutorial within the shared
-      ``realm-tutorial-dotnet`` project. Specifically, you will be working in 
-      the following files:
-
-      .. cssclass:: config-table
-      .. list-table::
-         :header-rows: 1
-         :widths: 35 65
-
-         * - File
-           - Purpose
-
-         * - Item.cs (in the ``Models`` folder)
-           - Defines the Realm object we store in the database.
-
-         * - ItemPage.xaml.cs
-           - Provides the functionality for viewing, updating, and deleting items 
-             in the user's todo list.
+      organized. This is a organized like a standard MAUI 
+      `MVVM <https://learn.microsoft.com/en-us/dotnet/maui/xaml/fundamentals/mvvm>`__ 
+      solution, with a single project containing the views, models, and view 
+      models. 
+      
+      The app uses a single model, ``Item``, which implements ``IRealmObject``.  
+      We have three views, one for logging in (``LoginPage``), another for 
+      viewing Items (``ItemsPage``), and a third for editing and creating new 
+      items. Each view has a corresponding view model. 
 
-         * - NewItemPopup.xaml and NewItemPopup.xaml.cs
-           - Provides the functionality to add a new item to the list.
+      In addition to the standard MVVM structure, we have centralized all of the 
+      Realm logic into a RealmService class, which is found in the "Services" folder. 
+      This architecture ensures we are sharing the same realm throughout. 
 
    .. step:: Run the App
 
       Without making any changes to the code, you should be able to run the app 
       in either the Android emulator or the iOS Simulator, or on a physical device.
-      When you installed the template app, the Realm CLI also set up a new 
+      
+      You don't need to make any changes because, when you installed the template 
+      app, the Realm CLI also set up a new 
```

Comment:
This is no longer quite accurate, because the user may have got the app from the UI .zip download or by git clone. I think you could just say something like: When you set up the template in the App Services UI or with the CLI, Atlas also set up a new backend ...

Reply:
@cbush please check this change.